src/components/AllArticles.jsx

```diff
@@ -9,15 +11,14 @@ const AllArticles = () => {
    useEffect(() => {
       getAllArticles()
       .then((articlesFromApi) => {
-         console.log(articlesFromApi);
          setArticles(articlesFromApi)
       })
    }, [])
 
    return (
       <div>
          <h2>All Articles</h2>
-         <ul>
+         <ul className="article-list">
             {articles.map((article, i) => {
```

Comment:
Every article has an id, it would be better to use article_id as the unique key to avoid future problems